src/Adapter/PlatformServices.Desktop/Utilities/FileUtility.cs

```diff
@@ -293,9 +293,15 @@ internal virtual string[] GetDirectoriesInADirectory(string directoryPath)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Requirement is to handle all kinds of user exceptions and message appropriately.")]
         private string GetSymbolsFileName(string path)
         {
+            Debug.Assert(!string.IsNullOrEmpty(path), "path");
+
             try
             {
-                return DiaHelper.GetSymbolsFileName(path);
+                string pdbFile = Path.ChangeExtension(path, ".pdb");
```

Comment:
From msdn, this would bomb if path has invalid chars: https://msdn.microsoft.com/en-us/library/system.io.path.changeextension(v=vs.110).aspx
Are we checking for them some place?